src/components/AddBook.tsx

```diff
@@ -17,9 +19,12 @@ interface IBookProps {
 
 const AddBook: FC<IBookProps> = ({ id, title, authors, image, description, isbn, publishedDate, categories, quantity }) => {
   const globalUser = useUser();
+  const [err, setErr] = useState<PostgrestError | null>();
   // eslint-disable-next-line
   const Add = async () => {
```

Comment:
I'd suggest naming it in lower-case. If it was not possible, try to change the name to a more descriptive one, like:
```suggestion
  const addToStore = async () => {
```